astroquery/linelists/cdms/data/catdir.cat

```diff
@@ -1,1089 +1,1198 @@
-   TAG |  NAME                  |      NLINE   |  lg(Q(1000))  |  lg(Q(500))   |  lg(Q(300))  |   lg(Q(225))  |   lg(Q(150))   |   lg(Q(75))  |  lg(Q(37.5))   |  lg(Q(18.75))  |  lg(Q(9.375)) |   lg(Q(5.000))  |  lg(Q(2.725))
```

Comment:
There has been a lot of back and forth with this file, please squash those changes out into one commit touching the datafile.

Reply:
can we just squash at the end?